src/_includes/styles.css

```diff
@@ -111,24 +112,92 @@ nav.header {
     position: -webkit-sticky;  /* because Safari is a laggard - 4/2019 */
     position: sticky;
     top: 0;
-    padding: 0.5rem;
+    padding: 5px 2px 4px 2px;
     margin-top: -1px; /* fix sunbeam through the blinds flicker */
-    padding-top: calc(0.5rem + 1px); 
+    max-height: calc(1.6em + 19px);
+    overflow: hidden;
+    transition: max-height 200ms;
 }
-@media (min-width: 500px) {
-    nav.header {
-        padding: 0.8rem;
-        padding-top: calc(0.8rem + 1px); /* fix sunbeam through the blinds flicker */
+nav.header.expanded {
+    max-height: 170px;
+}
+@media (min-width: 330px) {
+    nav.header.expanded {
+        max-height: 130px;
     }
 }
+@media (min-width: 430px) {
+    nav.header.expanded {
+        max-height: 90px;
+    }
+}
+
 ul.nav-buttons {
     list-style: none;
+    margin: 0 auto;
     padding: 0;
-    margin: 0;
     display: flex;
     flex-wrap: wrap;
-    justify-content: center;
     align-items: center;
+    justify-content: space-around;  /* space-evenly looks a little better, but doesn't work in IE11 or Edge */
```

Comment:
If we'd like to provide the niceties of `space-evenly` for browsers that support it, but fallback to `space-around` for ie11, we can just include it first and IE will ignore `space-evenly` since it doesn't recognize it:

See 5c410f70178576f6cf7c59ae110d1cb14df56979 - add ie 11 compatibility header and justify-content fallback

Reply:
When I tried a fallback like that it worked well for IE11, but not for Edge.  Then I realized that space-around looked fine and figured maybe just stick with that.